### PR TITLE
ACL doc improvements

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

**Comment by nickanderson** [comment]
@craigcomstock can you review please?

File: reference/promise-types/files.markdown [hunk 1]
```diff
@@ -319,34 +319,57 @@ Form of the permissions is as follows:
...
+** The groups id is not a valid alternative.
+** This ACL is **required** when `acl_method` is set to `overwrite`.
 
 * `gid`
 
 A valid group identifier for the system and cannot be empty. However, in some
 ACL types, `gid` can be set to `*` to indicate a special group (e.g. in POSIX
 this refers to the file group).
 
+**Note:** The group name is not a valid alternative.
```
Comment (craigcomstock) [inline reference/promise-types/files.markdown]: I think group name is fine. I'm not sure that gid works.

Comment (craigcomstock) [reply reference/promise-types/files.markdown]: oh, sorry, gid should be an id, group should be a group-name. got it.

Comment (nickanderson) [reply reference/promise-types/files.markdown]: OK, feel free to suggest different wording that mightmake it more clear

**Pull request merged**